lib/appsignal_phoenix/live_view.ex

```diff
@@ -18,21 +19,20 @@ defmodule Appsignal.Phoenix.LiveView do
           kind, reason ->
             stack = __STACKTRACE__
 
+            @tracer.set_params(MapFilter.filter(params))
+            @tracer.set_environment(Appsignal.Metadata.metadata(socket))
```

Comment:
I think this looks fine. We can't chain it anymore on the span, like we did previously below, but is that necessary?

But maybe @jeffkreeftmeijer has a stronger opinion about this?